src/server/db/queries.ts

```diff
@@ -191,3 +192,22 @@ export async function getFirstOrganizationForUser(userId: string) {
         .limit(1)
         .executeTakeFirst()
 }
+
+export const getUsersByRoleAndOrgId = async (role: 'researcher' | 'reviewer', orgId: string) => {
+    let query = db
+        .selectFrom('user')
+        .innerJoin('orgUser', 'user.id', 'orgUser.userId')
+        .innerJoin('org', 'orgUser.orgId', 'org.id')
+        .selectAll()
+        .where('orgUser.orgId', '=', orgId)
+
+    if (role === 'researcher') {
+        query = query.where('orgUser.isResearcher', '=', true)
+    }
+
+    if (role === 'reviewer') {
+        query = query.where('orgUser.isReviewer', '=', true)
+    }
+
+    return await query.execute()
```

Comment:
https://kysely.dev/docs/recipes/expressions#conditional-expressions says not to do this, instead use filters

Reply:
i like the 2nd approach linked for sure
```
  .where((eb) => {
    const filters: Expression<SqlBool>[] = []
```

but i think its important to note that both approaches are valid and they dont say to not use it:
```
Having a set of optional filters you want to combine using and, is the most basic and common use case of conditional where expressions. Since the where, having and other filter functions are additive, most of the time this is enough:
```